system_tests/fast_confirm_test.go

```diff
@@ -40,9 +44,109 @@ import (
 	"github.com/offchainlabs/nitro/validator/valnode"
 )
 
+func TestFastConfirmationWithdrawal(t *testing.T) {
+	ctx, cancelCtx := context.WithCancel(context.Background())
+	defer cancelCtx()
+	builder, stakerA, cleanupBuilder, cleanupBackgroundTx := setupFastConfirmation(ctx, t)
+	defer cleanupBuilder()
+	defer cleanupBackgroundTx()
+
+	// Withdraw ETH from L2 to L1
+	gethhook.RequireHookedGeth()
+	arbSys, err := precompilesgen.NewArbSys(types.ArbSysAddress, builder.L2.Client)
+	Require(t, err)
+	authL2 := builder.L2Info.GetDefaultTransactOpts("User", ctx)
+	authL2.Value = big.NewInt(1000)
+	tx, err := arbSys.WithdrawEth(&authL2, common.Address{})
+	Require(t, err, "ArbSys failed")
+
+	// Wait for transaction to be executed
+	time.Sleep(4 * time.Millisecond) // Geth takes a few ms for the receipt to show up
+	var receipt *types.Receipt
+	receipt, err = builder.L2.Client.TransactionReceipt(ctx, tx.Hash())
+	Require(t, err, "No receipt for txn")
+	if receipt.Status != types.ReceiptStatusSuccessful {
+		Fatal(t, "Tx failed with status code:", receipt)
+	}
```

Comment:
Why not using EnsureTxSucceeded?

```suggestion
	receipt, err := builder.L2.EnsureTxSucceeded(tx)
	Require(t, err)
```